### ISSUE TITLE
Breakpoint list appears empty when opening VSCode and saved breakpoints exist. 

### ISSUE THREAD

**Issue by MRWeather** [PowerShell/vscode-powershell#1093]
## Issue Description ##
Breakpoint list appears empty when opening VSCode and saved breakpoints exist. The Deactivate and Remove all tool buttons are enabled, indicating that there are breakpoints but the actual breakpoints don't show until debugging is started or the add or deactivate buttons are clicked. 
Steps to reproduce:
1. Open VSCode and a powershell file
2. Set one or more breakpoints
3. Close VSCode
4. Reopen VSCode
5. Click Debug tab (or press Ctrl+Shift+D)
6. Click on Breakpoints to expand - no breakpoints are shown.
7. Click Deactivate Breakpoints - breakpoints are shown, deactivated
## Attached Logs ##

Follow the instructions in the [README](https://github.com/PowerShell/vscode-powershell#reporting-problems) about capturing and sending logs.

### Visual Studio Code ###

| Name | Version |
| --- | --- |
| Operating System | Windows_NT x64 10.0.15063 |
| VSCode | 1.17.2|
| PowerShell Extension Version | 1.5.0 |

### PowerShell Information ###

|Name|Value|
|---|---|
|PSVersion|5.1.15063.674|
|PSEdition|Desktop|
|PSCompatibleVersions|1.0 2.0 3.0 4.0 5.0 5.1.15063.674|
|BuildVersion|10.0.15063.674|
|CLRVersion|4.0.30319.42000|
|WSManStackVersion|3.0|
|PSRemotingProtocolVersion|2.3|
|SerializationVersion|1.1.0.1|

### Visual Studio Code Extensions ###

<details><summary>Visual Studio Code Extensions(Click to Expand)</summary>

|Extension|Author|Version|
|---|---|---|
|PowerShell|ms-vscode|1.5.0|;

</details>

**Comment by DarkLite1** [comment]
Seems to work fine here on:
VS Code: 1.19.0-insider
PS Extension: 1.5.0

Do you have the same problem after an update of VS Code?

**Comment by rkeithhill** [comment]
Did you start a `PowerShell Interactive Session" debug session?  This is where I see the problem.  I believe who two rights making a wrong.  We added code to clear breakpoints in the interactive session when a debug session starts so we could start from a "known" state.  At about the same time, code was added to update VSCode about breakpoint changes a user made in the console. I think the combo winds up wiping out the breakpoints in VSCode.  I "think" this should be an easy fix.
